controllers/dashboard/routes.controller.js

```diff
@@ -129,6 +129,7 @@ function RoutesController(options) {
       }
     ], function(err, result) {
       if (err)
+        // TODO: this wont work, self is not defined
         return self.destroy(err);
```

Comment:
@jgigault Not sure what to do here. Can you make the appropriate change and merge ? I supposer the error `err` has to be handled somehow.